index.html

```diff
@@ -76,47 +76,45 @@
+        <p>Systers, an Anita Borg Institute (ABI) Community, has very specific needs with managing volunteers. We have the Volunteer Management System and Systers Portal to help Systers and volunteers to make an engaged connection. We also maintain the Systers GNU Mailman and apply our customized features to the latest release. Our customized version, is very complex and has a higher difficulty level for developers. We’re also working on a new project to help manage our community's meetups.</p>
 
         <table class="projects">
             <tr><td><a href="https://github.com/systers/vms/wiki" target="_blank">Volunteer Management System (Python w/Django)</a></td>
-                <td><a href="https://github.com/systers/portal/wiki" target="_blank">Systers Portal(Python w/Django)</a></td></tr>
+                <td><a href="https://github.com/systers/portal/wiki" target="_blank">Systers Portal (Python w/Django)</a></td></tr>
             <tr><td><a href="https://github.com/systers/automated-testing/wiki" target="_blank">Automated Testing (Selenium using Java & Python)</a></td>
-                <td><a href="https://github.com/systers/ossprojects/wiki/Meetup%20Features" target="_blank">Meetup Features for Portal</a></td></tr>
-            <tr><td><a href="https://github.com/systers/ossprojects/wiki/Systers%20Features%20for%20Mailman%203" target="_blank">Systers Features for Mailman 3</a></td>
-                <td><a href="https://github.com/systers/ossprojects/wiki/PowerUp" target="_blank">PowerUp</a></td></tr>
+                <td><a href="https://github.com/systers/ossprojects/wiki/Meetup%20Features" target="_blank">Meetup Features for Portal (Python w/Django)</a></td></tr>
+            <tr><td><a href="https://github.com/systers/ossprojects/wiki/Systers%20Features%20for%20Mailman%203" target="_blank">Systers Features for Mailman 3 (Python w/Django)</a></td>
+                <td><a href="https://github.com/systers/ossprojects/wiki/PowerUp" target="_blank">PowerUp (Android/iOS)</a></td></tr>
         </table>
 
     <div class="subheader" id="programs"><h2>Programs</h2></div>
-        <p>Systers participates in several Open Source Programs, including <a href="https://summerofcode.withgoogle.com/" target="_blank">Google Summer of Code</a>, <a href="https://www.outreachy.org/" target="_blank">Outreachy</a>, and various codeathons to encourage new people to contribute to our projects and become part of our community.</p>
+        <p>Systers participates in several Open Source Internships, including <a href="https://summerofcode.withgoogle.com/" target="_blank">Google Summer of Code</a>, <a href="https://www.outreachy.org/" target="_blank">Outreachy</a>, and <a href="https://railsgirlssummerofcode.org/" target="_blank">Rails Girls Summer of Code</a> to encourage new people to contribute to our projects and become part of our community.</p>
```

Comment:
These are not all internships and we are not currently participating in RGSoC, please remove these changes.